packages/components/src/globals/scss/_tooltip.scss

```diff
@@ -224,6 +224,39 @@
   $caret-width: rem(8px);
   $body-spacing: $caret-spacing + $caret-height;
 
+  // Use pseudo element to create invisible hover area to keep tooltip open on hover
+  .#{$prefix}--assistive-text::after {
+    position: absolute;
+    display: block;
+    content: '';
+    // clip-path: polygon(50% 100%, 0 0, 100% 0);
```

Comment:
Will remove before the merge, but this is what we'd need to use if we want a chevron hover area instead of a rectangle

Reply:
The rectangle works great!

If you ever do decide to go with a chevron-ish shape, please don't go with a "pointy tail" - perhaps something with a "flat tail" that is the same width as the button or icon?

![image](https://user-images.githubusercontent.com/3331913/111709263-a8104600-881d-11eb-91dd-04cc59d00ae5.png)